runtime/lua/vim/_meta/lpeg.lua

```diff
@@ -69,7 +88,7 @@ function Pattern:match(subject, init) end
 
 --- Returns the string `"pattern"` if the given value is a pattern, otherwise `nil`.
 ---
---- @return 'pattern'|nil
+--- @return "pattern"|nil
```

Comment:
note this workaround; for the bug https://github.com/neovim/neovim/pull/26989#discussion_r1450986576